plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -83,11 +83,11 @@ public class PluginConfiguration extends OkaeriConfig {
     public boolean updateInfo = true;
 
     @Comment("Czy informacje o aktualizacji wersji nightly maja byc widoczne podczas wejscia na serwer")
-    @Comment("Ta opcja działa tylko wtedy, gdy także jest włączona opcja 'update-info'")
+    @Comment("Ta opcja dziala tylko wtedy, gdy takze jest wlaczona opcja 'update-info'")
     public boolean updateNightlyInfo = true;
 
-    @Comment("Czas w godzinach. o który ma zostać przesunięty czas w np. placeholderach. Przyjmuje zarówno wartości dodatnie jak i ujemne")
-    @Comment("Opcja przydatna, kiedy strefa czasowa ustawiona na serwerze jest inna niz w kraju docelowym (np. serwer ma ustawiona strefe czasowa \"Europe/London\", a gracze są z Polski - wystąpi wtedy godzina różnicy czasu)")
+    @Comment("Czas w godzinach, o ktory ma zostac przesuniety czas w np. placeholderach. Przyjmuje zarowno wartosci dodatnie jak i ujemne")
+    @Comment("Opcja przydatna, kiedy strefa czasowa ustawiona na serwerze jest inna niz w kraju docelowym (np. serwer ma ustawiona strefe czasowa \"Europe/London\", a gracze sa z Polski - wystapi wtedy godzina roznicy czasu)")
```

Comment:
Czemu się pozbywamy polskich znaków? Seems like a downgrade.

Reply:
Because for some reason, it's break formatowanie even if JVM has ustawiony odpowiedni start argument.

Reply:
???????????

Reply:
I'm literally using it here with no problems: https://github.com/OkaeriPoland/okaeri-minecraft/blob/master/openvote/shared/src/main/java/eu/okaeri/minecraft/openvote/shared/OpenVoteConfig.java

![image](https://user-images.githubusercontent.com/44551064/173183200-31e9d5c7-0fd9-4212-88cc-ef868b0280b9.png)

Reply:
Diacritics added to all configuration comments